core/modes/language_modes.py

```diff
@@ -57,12 +70,12 @@
     "css": ["c s s"],
     "gdb": ["g d b"],
     "go": ["go", "go lang", "go language"],
-    "r": ["are language"],
+    "r": ["are lang", "are language"],
+    "scm": ["scheme", "s c m", "tree sitter"],
```

Comment:
I don't think this fulfills the criteria `Only put things
# here which have a supported language mode; that's why there are so many`

Reply:
This entire list is also going away in this pull request
https://github.com/talonhub/community/pull/1528

Reply:
Removed